HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /KasRoudra/PyPhisher/main/files/version.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f4692625db0>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=None)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /KasRoudra/PyPhisher/main/files/version.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f14b2d1dc30>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=None)'))
HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /KasRoudra/PyPhisher/main/files/version.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fbf63cbabc0>, 'Connection to raw.githubusercontent.com timed out. (connect timeout=None)'))
